index.html

```diff
@@ -1,228 +1,65 @@
-                <td>
-                  <a class="link" href="https://github.com/hicsail"><i class="fa fa-github"></i></a>
-                  <a class="link" href="https://www.linkedin.com/company/boston-university-software-&-application-innovation-lab"><i class="fa fa-linkedin-square"></i></a>
-                  <a class="link" href="https://twitter.com/hicsail"><i class="fa fa-twitter-square"></i></a>
-                  <a class="link" href="mailto:hicsail@bu.edu"><i class="fa fa-envelope-o"></i></a>
-                  <a class="link" href="https://maps.bu.edu/?id=647#!m/330601"><i class="fa fa-map-marker"></i></a>
-                </td>
-              </tr>
-            </table>
-          </div> -->
-          <footer class="footer-container">
-            <img id = "footer-img" onclick="location.href='https://www.bu.edu/hic';" src="//sail.codes/img/hic.png" alt="Hariri Institute for Computing"/>
-            <ul class="social-icons">
-              <li><a class="link" href="https://github.com/hicsail"><i class="fa fa-github"></i></a></li>
-              <li><a class="link" href="https://www.linkedin.com/company/boston-university-software-&-application-innovation-lab"><i class="fa fa-linkedin-square"></i></a></li>
-              <li><a class="link" href="https://twitter.com/hicsail"><i class="fa fa-twitter-square"></i></a></li>
-              <li><a class="link" href="mailto:hicsail@bu.edu"><i class="fa fa-envelope-o"></i></a></li>
-              <li><a class="link" href="https://maps.bu.edu/?id=647#!m/330601"><i class="fa fa-map-marker"></i></a></li>
-            </ul>
-            <!-- </div> -->
-        </footer>  
-        <!-- </footer> -->
-      </div>
-    </div>
-  </div>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/4.3.0/d3.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/hammer.js/2.0.8/hammer.js"></script>
-  <script src="js/hammerPlugin.js"></script>
-  <script src="js/post.js"></script>
-  <script src="js/astroids.js"></script>
-</body>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>SAIL</title>
+
+        <!-- <style>
+      html {
+        background-color: white;
+      }
+    </style> -->
+
+        <!-- favicon -->
+        <link rel="icon" type=”image/png href="/favicon/s_logo.png">
+
+        <!-- alternative favicon -->
+        <!-- <link
+      rel="alternate icon"
+      type="image/png"
+      href="/favicon/**.png"
+    /> -->
+
+        <link rel="apple-touch-icon" href="/favicon/s_logo.png" />
+        <link rel="manifest" href="/favicon/site.webmanifest" />
+        <meta name="theme-color" content="#000000" />
+
+        <!-- Start Single Page Apps for GitHub Pages -->
+        <script src="js/astroids.js"></script>

```

Comment:
Do we still need this file if you've ported it to React?

Reply:
It isn't necessary any longer